.pre-commit-config.yaml

```diff
@@ -0,0 +1,33 @@
+# To use these pre-commit hooks you should first install http://pre-commit.com
+# and then run "pre-commit install" in the root folder of your local clone
+# so the hooks are deployed to .git/hooks
+#
+# See http://pre-commit.com for more information
+# See http://pre-commit.com/hooks.html for more hooks
+#
+# the pep8 custom config (e.g. custom line lenght) is defined in file .pep8
+# http://pep8.readthedocs.io/en/release-1.7.x/intro.html#configuration
+
+repos:
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    sha: v1.2.0
+    hooks:
+      - id: autopep8-wrapper
+        name: 'autopep8-wrapper (.eb)'
+        types: [file]
+        files: \.eb$
+      - id: check-ast
+        name: 'check-ast (.eb)'
+        types: [file]
+        files: \.eb$
+      - id: autopep8-wrapper
+      - id: check-ast
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-merge-conflict
+
+  - repo: https://github.com/pescobar/pre-commit-hooks-easybuild
+    sha: master
```

Comment:
Sorry about the drive-by review -- was just doing some googling and came across this.

Note that branch names aren't going to be repeatable and so [they're discouraged](https://pre-commit.com/#using-the-latest-sha-for-a-repository)

Reply:
@asottile Your feedback is much appreciated!